qiskit/algorithms/minimum_eigen_solvers/qaoa/qaoa.py

```diff
@@ -127,34 +119,30 @@ def __init__(self,
         validate_min('p', p, 1)
 
         self._p = p
-        self._mixer = mixer.to_opflow() if isinstance(mixer, LegacyBaseOperator) else mixer
+        self._mixer = mixer
         self._initial_state = initial_state
 
         # VQE will use the operator setter, during its constructor, which is overridden below and
```

Comment:
I do not think this prior comment is valid any more

Reply:
I removed the comment